### ISSUE TITLE
False first_seen_date in clients_first_seen

### ISSUE THREAD

**Issue by jmccrosky** [mozilla/bigquery-etl#1042]
There appears to be an error in the calculation of the first_seen_date field of the clients_first_seen table.

For example:

```
SELECT
  client_id,
  first_seen_date
FROM
  `moz-fx-data-shared-prod.telemetry.clients_first_seen`
WHERE
  client_id = "6aec81c8-0eb5-4bc7-8480-bc2b5dc76322"
```

Gives a first_seen_date of 2020-06-02.

While,

```
SELECT
  client_id,
  submission_date
FROM
  `moz-fx-data-shared-prod.telemetry.clients_daily`
WHERE
  client_id = "6aec81c8-0eb5-4bc7-8480-bc2b5dc76322"
  AND submission_date <= "2020-06-02"
```

Shows many submission dates well before that date.

This problem does appear to happen frequently.

**Comment by jklukas** [comment]
It looks like this was due to an error I made in manually backfilling the table after the last change (#1024). The backfill was successful, but then we made some final changes to the query in review, and I reran two days to test the updated query. During that last backfill operation I accidentally specified dates in 2019 rather than 2020, so dropped all data between 2019-06-01 and 2020-06-02.

We caught this just in time. I was able to restore the table to the state as of right before the bad backfill operation (BQ stores 7 days of change history for each table, and we're currently at 6 days). I am rerunning queries for the past 7 days now to get this all caught up. If we had caught this later in the week, we would have had to recreate the table from scratch, which would have been significantly more expensive.
